### PR TITLE
upgrade ant version

### PR DESCRIPTION
**Pull request by mmusgrov**
Bring the version of ant up to date.

### CODE REVIEW

File: integration/ejb/perftest/perf-eap5/pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.8.2</version>
+            <version>[4.13.1,)</version>
```
Comment (mayankkunwar) [inline integration/ejb/perftest/perf-eap5/pom.xml:26]: Can we define the version in property, here and other modified pom.xml files?

**Comment by mayankkunwar** [comment]
@mmusgrov now CI build is taking more time to complete the job. This time it tried to exceed 900 minutes time limit, that's why it failed.

File: integration/ejb/perftest/perf-eap5/pom.xml (hunk 1 above)
Comment (mmusgrov) [reply integration/ejb/perftest/perf-eap5/pom.xml:26]: No because they don't all have parents (only one of them did).

**Comment by mmusgrov** [comment]
> @mmusgrov now CI build is taking more time to complete the job. This time it tried to exceed 900 minutes time limit, that's why it failed.

Okay so we need to look into that.
But this PR can still be merged once you're okay with the changes.

File: integration/ejb/perftest/perf-eap5/pom.xml (hunk 1 above)
Comment (mayankkunwar) [reply integration/ejb/perftest/perf-eap5/pom.xml:26]: Okay

Comment (mmusgrov) [reply integration/ejb/perftest/perf-eap5/pom.xml:26]: If you want them all to inherit from a common parent where we would defined common properties you would need to raise a separate JBTM for that,

**Review by mayankkunwar — APPROVED** [review]

**Pull request merged**